File: LeetCode/JavaScript/#34 Search for a Range.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var searchRange = function(nums, target) {
    var res = [-1,-1];
    var pos = nums.indexOf(target);
    if(pos===-1) return res;
    res[0] = pos;
    pos =nums.length-(1 + nums.reverse().indexOf(target));
    res[1] = pos;
    return res;
};

File: LeetCode/Python/#34 Search for a Range.py
class Solution(object):
    def searchRange(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        res = [-1,-1]
        cnt = nums.count(target)
        if cnt==0:
            return res
        end = nums.index(target)
        return [end,end+cnt-1]